contracts/contracts/vault/VaultCore.sol

```diff
@@ -80,7 +72,7 @@ contract VaultCore is VaultStorage {
         emit Mint(msg.sender, priceAdjustedDeposit);
 
         // Rebase must happen before any transfers occur.
-        if (unitAdjustedDeposit >= rebaseThreshold && !rebasePaused) {
```

Comment:
I think we can remove this check because the entire function is blocked by a paused check.

Reply:
This means I'm still missing something. The variable should already be renamed so that code shouldn't be valid I think.

I'll check later.